File: Python/tests/test_cra_exp.py
import pytest

from hbayesdm.models import cra_exp


def test_cra_exp():
    _ = cra_exp(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_cra_exp.R
context("Test cra_exp")
library(hBayesDM)

test_that("Test cra_exp", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(cra_exp(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
